.github/ISSUE_TEMPLATE/test-scenario.md

```diff
@@ -2,7 +2,7 @@
 name: Test Scenario
```

Comment:
We can add a placeholder for Bug identified performing this test

Reply:
Sorry, I'm not sure what you mean. Can you expand on this?

Reply:
I have made the changes